lib/Crust/Middleware/Static.pm6

```diff
@@ -51,6 +51,6 @@ method !handle-static(%env) {
     }
 
     temp %env<PATH_INFO> = $path;
-    my @res = $!file.(%env);
+    my @res = await $!file.(%env);
```

Comment:
`$!file.(%env)` should be a Promise, so just returning `$!file.(%env)` seems enough.